assignments/callbacks.js

```diff
@@ -25,27 +25,63 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
+function gettingLength(arr, cb){
+  return cb(arr);
   // getLength passes the length of the array into the callback.
 }
 
+function itemsLength(items){
+  return items.length;
+}
+console.log(gettingLength(items, itemsLength));
+
 function last(arr, cb) {
+  return cb(arr);
   // last passes the last item of the array into the callback.
 }
 
+function lastArr(items){
+  return items.length - 1;
+}
+
+// console.log(last(items, lastArr));
+
+
 function sumNums(x, y, cb) {
+  return cb(x + y);
   // sumNums adds two numbers (x, y) and passes the result to the callback.
 }
 
+function add(num1, num2){
+  return num1 + num2;
+}
+
+console.log(add(3,4));
+
 function multiplyNums(x, y, cb) {
+  return cb(x * y);
   // multiplyNums multiplies two numbers and passes the result to the callback.
 }
+function timesTwo(num1, num2) {
+  return num1 * num2; 
+}
+console.log(timesTwo(2,8));
+
 
 function contains(item, list, cb) {
+  return cb(list,item);
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
 }
 
+function present(items, list){
+ return 
+}
```

Comment:
check out mdn for help on this one. There is an includes method you might be able to use here to check the array for the item that is passed as an argument.